dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/sms/config/SmsMessageSender.java

```diff
@@ -122,7 +123,7 @@ public OutboundMessageResponse sendMessage(
   public Future<OutboundMessageResponse> sendMessageAsync(
       String subject, String text, String footer, User sender, Set<User> users, boolean forceSend) {
     OutboundMessageResponse response = sendMessage(subject, text, footer, sender, users, forceSend);
-    return new AsyncResult<>(response);
+    return CompletableFuture.completedFuture(response);
```

Comment:
This is doing nothing, it is not running sendMessage in a different thread and it is not running it in parallel.
Would it make sense to just return the response?